### DISCUSSION TITLE
commit 4188b22a adds ﻿▸ in agnoster theme

### DISCUSSION THREAD

**Discussion by thanosz** [ohmyzsh/ohmyzsh#12449] [Help] [answered]
Hello, as per title, commit 4188b22a adds a ▸ (9656 25B8 BLACK RIGHT-POINTING SMALL TRIANGLE) in the beggining of agnoster theme

<img width="778" alt="image" src="https://github.com/ohmyzsh/ohmyzsh/assets/8742206/ee6b0a82-8e93-4f6a-a619-a9d051bc6f20">

To reproduce, 
```
export ZSH_THEME="agnoster"
git checkout c189e8b4
zsh
git checkout 4188b22a
zsh
```

Was this intented?

**Comment by carlosala** [comment, marked as answer]
Hi! I can't reproduce it and don't see that symbol in the whole repo. Make sure you have no changes in the repo, otherwise might be an issue of your terminal emulator.

**Reply by thanosz** [reply]
Hello. I confirmed that this happens with iterm2 locally.  Strangely the symbol appears with 4188b22a. Probably related with latest iterm 3.50 published 20-may. Thanks for taking a look!

**Reply by thanosz** [reply]
BTW, I just found it https://stackoverflow.com/questions/63063328/extraneous-blue-glyph-to-left-of-prompt
